x-pack/plugin/esql/qa/testFixtures/src/main/resources/stats.csv-spec

```diff
@@ -1947,8 +1947,7 @@ MIN(i):integer | a:keyword | b:integer
 // end::multi-mv-group-result[]
 ;
 
-statsByConstant
-required_capability: stats_by_constant
+statsByConstant#[skip:-8.14.1,reason:implemented in 8.14]
```

Comment:
@fang-xing-esql instead of skipping based on the new capability we added, I thought we'll get better coverage if we skip based on the version from which this was first possible.